core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -118,6 +118,26 @@ export class PickerColumnInternal implements ComponentInterface {
     }
   }
 
+  componentDidRender() {
+    const { activeItem, el, items, isColumnVisible, value } = this;
+
+    if (isColumnVisible) {
+      if (activeItem) {
+        if (el.scrollTop !== this.getScrollTopPositionForElement(activeItem)) {
+          // Scroll the active item into the view if after rendering
+          // it is still not visible
+          this.scrollActiveItemIntoView();
+        }
+      } else if (items[0].value !== value) {
+        // If the picker column does not have an active item and the current value
+        // does not match the first item in the picker column, that means
+        // the value is out of bounds. In this case, we assign the value to the
+        // first item to match the scroll position of the column.
+        this.value = items[0].value;
```

Comment:
Need to see where Ryan lands on the infinite loop/render Stencil PR. This works for now without any noticeable issues besides the developer warning from Stencil when programmatically assigning a value. 

This does not solve cases where developers assign a value that is less than the min or greater than the max. Since the date time does not enforce or sanitize values, they can still have values that are unavailable to the picker column. This really isn't a problem worth solving now, likely an implementation detail for implementers.